tests/outputs/a_expense/expenses.yaml

```diff
@@ -43,100 +49,81 @@
       day: 0
       month: tbd
       submission_day: 20
-      submission_month: jun
+      submission_month: 6
       submission_year: 2020
       where: bank
       year: 2020
   status: submitted
 test:
   expense_type: business
   grant_percentages:
-    - 50
-    - 50
+    - '50'
+    - '50'
   grants:
     - dmref15
-    - mrsec14
+    - SymPy-1.1
   itemized_expenses:
     - day: 1
-      month: jan
+      month: Jan
       year: 2018
       purpose: test
       unsegregated_expense: 10
       segregated_expense: 0
     - day: 2
-      month: jan
+      month: Jan
       year: 2018
       purpose: test
       unsegregated_expense: 20
       segregated_expense: 0
     - day: 3
-      month: jan
+      month: Jan
       year: 2018
       purpose: test
       unsegregated_expense: 30
       segregated_expense: 0
     - day: 4
-      month: jan
+      month: Jan
       year: 2018
-      purpose: 2
+      purpose: test
       unsegregated_expense: 40
       segregated_expense: 0
     - day: 5
-      month: jan
+      month: Jan
       year: 2018
       purpose: test
       unsegregated_expense: 50
       segregated_expense: 0
     - day: 6
-      month: jan
+      month: Jan
       year: 2018
       purpose: test
       unsegregated_expense: 60
       segregated_expense: 0
     - day: 7
-      month: jan
+      month: Jan
       year: 2018
       purpose: test
       unsegregated_expense: 70
       segregated_expense: 0
     - day: 8
-      month: jan
+      month: Jan
       year: 2018
       purpose: test
       unsegregated_expense: 80
       segregated_expense: 0
     - day: 9
-      month: jan
+      month: Jan
       year: 2018
       purpose: test
       unsegregated_expense: 90
       segregated_expense: 0
     - day: 10
-      month: jan
+      month: Jan
       year: 2018
-      purpsoe: test
+      purpose: test
       unsegregated_expense: 100
       segregated_expense: 0
+  overall_purpose: testing the databallectionsse
   payee: scopatz
-  project: cyclus
-  overall_purpose: testing for databallectionsse
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+  project: Cyclus
+  
```

Comment:
clean up end of file.  Not sure what the problem is here....

Reply:
done